src/main/java/jenkins/branch/MultiBranchProject.java

```diff
@@ -1350,9 +1339,11 @@ protected void execute(TaskListener listener) throws InterruptedException, IOExc
                 LOGGER.fine("Disabled. Skipping execution");
                 return;
             }
-
-            for (MultiBranchProject<?, ?> p
-                    : Jenkins.getInstance().getAllItems(MultiBranchProject.class)) {
+            Jenkins j = Jenkins.getInstance();
```

Comment:
Well `getInstance` is `@CheckForNull` whereas `getActiveInstance` is `@Nonnull`. So you use `getInstance` if you can gracefully recover, and `getActiveInstance` where you would do no better than to just throw an `IllegalStateException` yourself.